app/assets/javascripts/project_preview.coffee

```diff
@@ -0,0 +1,14 @@
+root = exports ? this
+
+root.sendPreview = ->
+  sform = $($('[data-js="project_form"]')[0].elements).not('input:hidden[name=_method]').serialize()
+  $.ajax
+    type: 'POST'
+    url: '/projects/preview'
```

Comment:
It's better to get the URL from a data attribute from within the form. The form (created from a view) can then use the Rails routes helper (probably `preview_projects_path`). This reduces the coupling between javascript and the responsible controller name.